app/build.gradle

```diff
@@ -24,13 +24,18 @@ android {
         buildConfigField 'String', 'LIGHTNING_VERSION_NAME', '"4.2.3.1"'
         buildConfigField "int", "MINIMUM_WEBVIEW_VERSION", "55"
 
-        if (this.hasProperty("com.cliqz.lumen.vpnApiKey")) {
-            buildConfigField 'String', 'VPN_API_KEY', this.getProperty("com.cliqz.lumen.vpnApiKey")
+        def vpnApiKey = this.hasProperty("com.cliqz.lumen.vpnApiKey") ?
+                this.getProperty("com.cliqz.lumen.vpnApiKey") : System.getenv("vpnApiKey")
+        if (vpnApiKey != null) {
+            buildConfigField 'String', 'VPN_API_KEY', '"' + vpnApiKey + '"'
         } else {
             buildConfigField 'String', 'VPN_API_KEY', '""'
         }
-        if (this.hasProperty("com.cliqz.lumen.revenuecatApiKey")) {
-            buildConfigField 'String', 'REVENUECAT_API_KEY', this.getProperty("com.cliqz.lumen.revenuecatApiKey")
+
+        def revenueCatApiKey = this.hasProperty("com.cliqz.lumen.revenueCatApiKey") ?
+                this.getProperty("com.cliqz.lumen.revenueCatApiKey") : System.getenv("revenueCatApiKey")
+        if (revenueCatApiKey != null) {
+            buildConfigField 'String', 'REVENUECAT_API_KEY', '"' + revenueCatApiKey + '"'
```

Comment:
`buildConfigField 'String', 'REVENUECAT_API_KEY', '"$(revenueCatApiKey)"'`

Reply:
```groovy
buildConfigField 'String', 'REVENUECAT_API_KEY', "\"${revenueCatApiKey}\""
```

you can not iterpolate strings literals starting with a single quote